src/track/frame.h

```diff
@@ -0,0 +1,102 @@
+#pragma once
+
+class QDebug;
```

Comment:
Do not forward declare types from external libraries. QDebug might not be a class, depending on how it is implemented.